publication-commons/build.gradle

```diff
@@ -3,7 +3,10 @@ repositories {
 }
 
 dependencies {
+    api project(":storage-model")
```

Comment:
moved from the bottom of the dependencies. Dependency management will be done at the end in separate PR